.vib/mediawiki/goss/mediawiki.yaml

```diff
@@ -0,0 +1,44 @@
+group:
+  daemon:
+    exists: true
+user:
+  daemon:
+    exists: true
+file:
+  /opt/bitnami/mediawiki:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /bitnami/mediawiki:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/mediawiki/images:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
+  /opt/bitnami/mediawiki/cache:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: daemon
+    group: root
```

Comment:
I don't really have a strong opinion on this, but in other cases where `-u "$WEB_SERVER_DAEMON_USER" -g "root"` are provided in the `configure_permissions_ownership`, we used `check_directories` skipping the uid:gid ownership check.

See Wordpress for example:
https://github.com/bitnami/containers/blob/5cd67cb7f1a11e143ad229b3fd266af55edb34fb/bitnami/wordpress/6/debian-11/rootfs/opt/bitnami/scripts/wordpress/postunpack.sh#L40-L44
https://github.com/bitnami/containers/blob/5cd67cb7f1a11e143ad229b3fd266af55edb34fb/.vib/wordpress/goss/vars.yaml#L11-L22


Cc. @FraPazGal @joancafom 

I don't have a strong opinion on whether to check ownership or not, but we should use the same criteria in all cases.

Reply:
Yes, I discussed this with @joancafom some time ago. To give some context, at the start we decided to avoid checking the owner/group to avoid extra checks, as we considered checking the permissions was enough. This was later revisited given that not checking `uid:gid` was not congruent with the idea of checking every change our compilation & postunpack logic applied on the asset.

I agree with you this will need to be replicated to the rest of the catalog but, given its scope, it's still in the air when this will be worked on. As we recently developed the tools to include `owner/group` in our Goss' templates, this approach wasn't enforced in our test suites still in development.

Reply:
Okay, in that case, I think we can proceed with merging these changes and then revisit it to extend to other assets.